arlo_server/__init__.py

```diff
@@ -841,6 +841,16 @@ def audit_report(election_id):
     report_writer.writerow(["Risk Limit", "{:d}%".format(election.risk_limit)])
     report_writer.writerow(["Random Seed", election.random_seed])
 
+    for audit_board in jurisdiction.audit_boards:
+        report_writer.writerow([
+            audit_board.name, audit_board.member_1,
```

Comment:
this highlights that an audit board name might not be the nice "Audit Board #1", and coud bite us later. OK for now, I don't have a better solution.

Reply:
I'm not sure what you mean - I was just going off the data model, which has a name field. What might bite us?

Reply:
I mean that if the audit board has a name of "Foobar", then the row in the CSV won't be clear that it's an audit board cause it has no title. Right now the audit boards have nice names, so it works okay.

For example, the previous line in the report says "Random Seed" explicitly.